doc/source/examples/analysis/custom_parallel_analysis.ipynb

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
<p>Here and everywhere else, do we need to filterwarning for this DCD reader deprecation?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Constantly running after warnings seems unsustainable, I'd be happy if the answer here is "it won't mess with CI, let's just keep it in because that's how it'll look to users" and we do nothing further on this one.

Reply:
<p>Currently I'm sanitizing <code>UserWarning</code> and  <code>DeprecationWarning</code>, so all the warning text get's replaced by something  like a <strong>USERWARNING</strong> when nbval evaluates it. This way, the CI doesn't fail due to repeated warnings, but it will still fail when a new warning shows up since they create the <code>stderr</code> attribute on the cell and if that wasn't present before, nbval will fail the CI. At the same time, we can't ignore the <code>stderr </code> .</p><p> </p><p>I don't know how often new warnings show up, either from MDA or some of its dependencies, but if it's a weekly thing, then yes, perhaps we should add a:</p><pre class="ql-syntax" spellcheck="false">import warnings
# suppress some MDAnalysis warnings
warnings.filterwarnings('ignore') 
</pre><p>on each notebook. Cells will still be tested and if they fail or their <code>stdout</code> changes nbval will catch them still.</p>

Reply:
<p>I'm going to leave this to a future problem and we can see how often CI fails.</p>